src/helpers/__tests__/LabelFilterHelper.test.ts

```diff
@@ -0,0 +1,226 @@
+  },
+  {
+    namespace: 'bookinfo',
+    healthPromise: new Promise(() => {}),
+    name: 'reviews',
+    istioSidecar: false,
+    labels: { app: 'reviews', service: 'reviews', version: 'v1,v2,v3' }
+  }
+];
+
+const workloadList: WorkloadListItem[] = [
+  {
+    namespace: 'bookinfo',
+    healthPromise: new Promise(() => {}),
+    name: 'details-v1',
+    type: 'Deployment',
+    istioSidecar: false,
+    labels: { app: 'details', version: 'v1' },
+    appLabel: true,
+    versionLabel: true
+  },
+  {
+    namespace: 'bookinfo',
+    healthPromise: new Promise(() => {}),
+    name: 'productpage-v1',
+    type: 'Deployment',
+    istioSidecar: false,
+    labels: { app: 'productpage', version: 'v1' },
+    appLabel: true,
+    versionLabel: true
+  },
+  {
+    namespace: 'bookinfo',
+    healthPromise: new Promise(() => {}),
+    name: 'ratings-v1',
+    type: 'Deployment',
+    istioSidecar: false,
+    labels: { app: 'ratings', version: 'v1' },
+    appLabel: true,
+    versionLabel: true
+  },
+  {
+    namespace: 'bookinfo',
+    healthPromise: new Promise(() => {}),
+    name: 'reviews-v1',
+    type: 'Deployment',
+    istioSidecar: false,
+    labels: { app: 'reviews', version: 'v1' },
+    appLabel: true,
+    versionLabel: true
+  },
+  {
+    namespace: 'bookinfo',
+    healthPromise: new Promise(() => {}),
+    name: 'reviews-v2',
+    type: 'Deployment',
+    istioSidecar: false,
+    labels: { app: 'reviews', version: 'v2' },
+    appLabel: true,
+    versionLabel: true
+  },
+  {
+    namespace: 'bookinfo',
+    healthPromise: new Promise(() => {}),
+    name: 'reviews-v3',
+    type: 'Deployment',
+    istioSidecar: false,
+    labels: { app: 'reviews', version: 'v3' },
+    appLabel: true,
+    versionLabel: true
+  }
+];
+
+const serviceList: ServiceListItem[] = [
+  {
+    namespace: 'bookinfo',
+    healthPromise: new Promise(() => {}),
+    name: 'details',
+    istioSidecar: false,
+    labels: { app: 'details', service: 'details' },
+    validation: { name: 'details', objectType: 'service', valid: true, checks: [] }
+  },
+  {
+    namespace: 'bookinfo',
+    healthPromise: new Promise(() => {}),
+    name: 'reviews',
+    istioSidecar: false,
+    labels: { app: 'reviews', service: 'reviews' },
+    validation: { name: 'reviews', objectType: 'service', valid: true, checks: [] }
+  },
+  {
+    namespace: 'bookinfo',
+    healthPromise: new Promise(() => {}),
+    name: 'ratings',
+    istioSidecar: false,
+    labels: { app: 'ratings', service: 'ratings' },
+    validation: { name: 'ratings', objectType: 'service', valid: true, checks: [] }
+  },
+  {
+    namespace: 'bookinfo',
+    healthPromise: new Promise(() => {}),
+    name: 'productpage',
+    istioSidecar: false,
+    labels: { app: 'productpage', service: 'productpage' },
+    validation: { name: 'productpage', objectType: 'service', valid: true, checks: [] }
+  }
+];
+
+describe('LabelFilter', () => {
```

Comment:
Nice, thanks!

Reply:
Removed, I removed it too in src/types/Filters.ts in the last commit...old code :)